@types/dom-parser/index.d.ts

```diff
@@ -0,0 +1,11 @@
+/// <reference types="node" />
```

Comment:
I think this file is unnecessary now, right?

Reply:
Yup, deleted.